Update language param to use SupportedLanguages enum

diondrapeck opened the pull request.

Jak-MS commented:
@lgayhardt 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

Note: the link issues in the validation comment should be addressed


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

diondrapeck commented:
> * Absolute link 'https://learn.microsoft.com/python/api/azure-ai-evaluation/azure.ai.evaluation.simulator.supportedlanguages?view=azure-python-preview' will be broken in isolated environments. Replace with a relative link.

Thanks, @Jak-MS - I've addressed the suggestions.

lgayhardt commented:
Thank you 

#sign-off
